### ISSUE TITLE
Remove duplicate lines in the logs

### ISSUE THREAD

**Issue by jean-christophe-manciot** [bit-team/backintime#833]
Ubuntu 17.10
bit 1.1.22

Example:
```
[I] Take snapshot (rsync: BACKINTIME: cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/management.properties -> /etc/java-8-openjdk/management/management.properties)
[C] cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/management.properties -> /etc/java-8-openjdk/management/management.properties
[I] Take snapshot (rsync: BACKINTIME: cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/snmp.acl -> /etc/java-8-openjdk/management/snmp.acl)
[C] cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/snmp.acl -> /etc/java-8-openjdk/management/snmp.acl
[I] Take snapshot (rsync: BACKINTIME: cd..t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/)
```
The fact that each line is duplicated makes it difficult to read; the following format would make it much easier:
```
[I] Take snapshot (rsync: BACKINTIME: cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/management.properties -> /etc/java-8-openjdk/management/management.properties)
[I] Take snapshot (rsync: BACKINTIME: cLc.t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/snmp.acl -> /etc/java-8-openjdk/management/snmp.acl)
[I] Take snapshot (rsync: BACKINTIME: cd..t...... media/actionmystique/SAMSUNG4-Ubuntu/chroot/sid/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/)
```
You probably should add a line after a rsync only if it fails.

**Comment by Germar** [comment]
They're only duplicates because you're using `Full rsync mode`.

With the old, default mode BiT run rsync twice. First with `--dry-run` where it will only log changes with leading `[C] ...`. If it found changes it will create hard-links and run rsync again. This time it will log what it has done with leading `[I] ...`

When we introduced `Full rsync mode` we kept both log lines because it would have been more irritating otherwise. BiT will filter them anyways when you browse the log through BiT.

But I agree with you when it comes to next upcoming version 1.2.0 because this will come without the old mode. Only `Full rsync mode` will be default. There will be no need anymore for both log lines. I'll remove it in 1.2.0
